ssh-uuid.sh

```diff
@@ -389,10 +389,8 @@ function get_rand_port_num {
 }
 
 function check_tool {
-	local tool
-	tool=$(which "$1")
-	if [[ ! -x "${tool}" ]]; then
-		quit "Unable to execute '${tool}'. Is it installed?"
+	if [ ! $(which "$1") ]; then
+		quit "'$1' not found in PATH. Is it installed?"
```

Comment:
Nice 👍  Before we merge the PR, would you mind squashing the two commits in a single one? I am a fan of clean commit histories. (I know not everybody agrees!)

Reply:
yes I'm up for it, I agree it's cleaner on the master branch then. done!